docs/guides/data-and-dbt-tests-report-ui.mdx

```diff
@@ -63,11 +66,13 @@ The rows are expandable, and on expansion show information about the last runs o
 - Affected rows
 - Time
 
-<img src="https://res.cloudinary.com/mintlify/image/upload/v1659304885/elementary/Test-Runs-video-demo_ubxssd.gif" alt="Test runs demo" />
+<img
+  src="https://res.cloudinary.com/mintlify/image/upload/v1659304885/elementary/Test-Runs-video-demo_ubxssd.gif"
+  alt="Test runs demo"
+/>
 
-## Configure Slack report
+## Exporting the report
 
-The data tests report UI can be sent via Slack when you run `edr monitor send-report`.
-Configure Slack integration by creating an app for Elementary and adding the `Token/Webhook` to the `config.yml`.
+The data tests report UI can be sent via Slack, Amazon S3, or Google Cloud Storage when you run `edr monitor send-report`.
```

Comment:
Elementary offers 3 methods that makes it easy to share the report with others:
1. Host on ...
2. Host on ... 
3. Send...

Refer to..."

I think that it's a bit useless to provide the command without instructions. 
Right? Better to refer or add the full explanation.

Reply:
What do you mean? I do that in the line below.
<img width="993" alt="image" src="https://user-images.githubusercontent.com/30181361/183853398-80726fae-d2a7-4c50-8f06-82beba9c0105.png">